src/FocusableView/FocusableView.tsx

```diff
@@ -1,14 +1,11 @@
 import * as React from "react";
 import * as cx from "classnames";
 import debounce = require("lodash/debounce");
-import { props, t, ReactChildren } from "../utils";
+import { props, t, ReactChildren, Children } from "../utils";
 
 export type FocusableViewRequiredProps = {
   /** FocusableView content. If a function it gets called with the boolean "focused" */
-  children:
-    | ((focused: boolean) => React.ReactNode | React.ReactNode[])
-    | React.ReactNode
-    | React.ReactNode[];
+  children: ((focused: boolean) => Children) | Children;
```

Comment:
`ReactNode[]` was superfluous (already part of `ReactNode` itself, it's recursive). Same goes for `Children`